### PR TITLE
Fix copyright in docs

### PR DESCRIPTION
**Pull request by znicholls**
Fixes the copyright and release version in the docs

 - [x] Fully documented, including `CHANGELOG.rst`

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -9,21 +9,29 @@
...
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
+import datetime
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+sys.path.append(os.path.join(os.path.dirname(__file__), "../../mesmer"))
```
Comment (mathause) [inline docs/source/conf.py:17]: Why are you setting these paths? mesmer should be installed?

File: docs/source/conf.py [hunk 2]
```diff
@@ -9,21 +9,29 @@
...
 
+from mesmer._version import get_versions
 
 # -- Project information -----------------------------------------------------
 
 project = "mesmer"
-copyright = "(c) 2021 ETH Zurich (Land-climate dynamics group, Prof. S.I. Seneviratne)"
-author = "Lea Beusch, Zebedee Nicholls, Mathias Hauser"
+copyright_year = datetime.date.today().year
+copyright = "(c) 2021-{} ETH Zurich (Land-climate dynamics group, Prof. S.I. Seneviratne), MESMER contributors listed in AUTHORS".format(copyright_year)
```
Comment (mathause) [inline docs/source/conf.py:27]:
```suggestion
current_year = datetime.date.today().year
copyright = f"(c) 2021-{current_year} ETH Zurich (Land-climate dynamics group, Prof. S.I. Seneviratne), MESMER contributors listed in AUTHORS"
```

**Review by mathause — APPROVED** [review]
LGTM!

File: docs/source/conf.py (hunk 2 above)
Comment (znicholls) [reply docs/source/conf.py]: Nice will add (sorry pushed before I saw this so can't commit the suggestion anymore)

File: docs/source/conf.py (hunk 1 above)
Comment (znicholls) [reply docs/source/conf.py]: I thought it was needed, turns out it is not

**Comment by mathause** [comment]
Thanks @znicholls

**Pull request merged**